api-docs/influxdb3/core/v3/ref.yml

```
@@ -0,0 +1,1168 @@
+      security:
+        - BearerAuth: []
+  /query:
+    get:
+      summary: Execute InfluxQL Query
+      description: Executes an InfluxQL query against the database.
+      parameters:
+        - name: db
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: q
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: format
+          in: query
+          required: false
+          schema:
+            type: string
+        - $ref: '#/components/parameters/Accept'
+      responses:
+        "200":
+          description: Successful query execution.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/QueryResponse'
+            text/csv:
+              schema:
+                type: string
+            application/vnd.apache.parquet:
+              schema:
+                type: string
+            application/jsonl:
+              schema:
+                type: string
+        "400":
+          description: Bad request.
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+        "403":
+          description: Access denied.
+        "404":
+          description: Database not found.
+        "405":
+          description: Method not allowed.
+        "422":
+          description: Unprocessable entity.
+      security:
+        - BearerAuth: []
+    post:
+      summary: Execute InfluxQL Query
+      description: Executes an InfluxQL query against the database.
+      parameters:
+        - $ref: '#/components/parameters/ContentType'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/QueryRequest'
+      responses:
+        "200":
+          description: Successful query execution.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/QueryResponse'
+            text/csv:
+              schema:
+                type: string
+            application/vnd.apache.parquet:
+              schema:
+                type: string
+            application/jsonl:
+              schema:
+                type: string
+        "400":
+          description: Bad request.
+        "401":
+          $ref: '#/components/responses/Unauthorized'
+        "403":
+          description: Access denied.
+        "404":
+          description: Database not found.
+        "405":
+          description: Method not allowed.
+        "422":
+          description: Unprocessable entity.
+      security:
+        - BearerAuth: []
+  /health:
+    get:
+      summary: Health Check
+      description: Returns the status of the service.
+      responses:
+        "200":
+          description: Service is running.
+        "500":
+          description: Service is unavailable.
+  /api/v1/health:
```

```suggestion
  /api/v1/health:
```
?
We don't use `/api/v1` for any other endpoints.